From 28f26283bae600341f880b9ef123a53a1eb56a31 Mon Sep 17 00:00:00 2001
From: Tim Lewis <elatllat@gmail.com>
Date: Thu, 23 Apr 2020 08:01:23 +0000
Subject: [PATCH 053/105] FROMLIST: arm64: dts: meson: S922X: extend cpu
 opp-points

HardKernel supports overclock through the addition to extra opp points
in their official kernel sources for odroid-n2 [1]. Christian tested
on other S922X devices with "no obvious issues". Neil thinks dvfs and
thermal managementis can keep other S922X devices safe.

[1] https://github.com/hardkernel/linux/commit/f86cd9487c7483b2a05f448b9ebacf6bd5a2ad2f
Tested-by: Christian Hewitt <christianshewitt@gmail.com>
Signed-off-by: Tim Lewis <elatllat@gmail.com>
---
 .../boot/dts/amlogic/meson-g12b-s922x.dtsi      | 17 +++++++++++++++++
 1 file changed, 17 insertions(+)

diff --git a/arch/arm64/boot/dts/amlogic/meson-g12b-s922x.dtsi b/arch/arm64/boot/dts/amlogic/meson-g12b-s922x.dtsi
index 046cc332d07f..d38f1f1a3bdf 100644
--- a/arch/arm64/boot/dts/amlogic/meson-g12b-s922x.dtsi
+++ b/arch/arm64/boot/dts/amlogic/meson-g12b-s922x.dtsi
@@ -65,6 +65,12 @@ opp-1896000000 {
 			opp-hz = /bits/ 64 <1896000000>;
 			opp-microvolt = <981000>;
 		};
+
+		opp-1992000000 {
+			opp-hz = /bits/ 64 <1992000000>;
+			opp-microvolt = <1001000>;
+		};
+
 	};
 
 	cpub_opp_table_1: opp-table-1 {
@@ -120,5 +126,16 @@ opp-1704000000 {
 			opp-hz = /bits/ 64 <1704000000>;
 			opp-microvolt = <891000>;
 		};
+
+		opp-1800000000 {
+			opp-hz = /bits/ 64 <1800000000>;
+			opp-microvolt = <981000>;
+		};
+
+		opp-1908000000 {
+			opp-hz = /bits/ 64 <1908000000>;
+			opp-microvolt = <1022000>;
+		};
+
 	};
 };
-- 
2.25.0

